docs/enterprise/maintenance.md

```diff
@@ -37,3 +37,9 @@ Private Packagist Enterprise installs a command line tool `packagist` on the hos
 #### Updating all packages
 
 Run the command `packagist package-update-all [--overwrite-data=true|false] [-only-non-updated=true|false]` to schedule update jobs on the worker for all packages on your installation. The flag `--overwrite-data` ensures that all existing version data is downloaded from the source again and version data is overwritten if it has changed. The flag `--only-non-updated` limits the set of packages to those which have only been initialized but never updated yet.
+
+### Migrate Private Packagist to another server
+
+Create a snapshot either via the Replicated admin console or using the CLI tool via `replicatedctl snapshot start` and either transfer the snapshot to the new machine or make sure it is accessible from the new machine.
```

Comment:
```suggestion
Create a snapshot either via the Replicated admin console or on the CLI with `replicatedctl snapshot start` and either transfer the snapshot to the new machine or make sure it is accessible from the new machine. We recommend to use SFTP on a remote machine or S3 to store snapshots, which can then also be accessed from a new server.
```

Reply:
So based on the support article here I am not sure that SFTP/S3 can be used for restore https://help.replicated.com/community/t/restoring-from-a-snapshot/68

Reply:
Actually looks like the screenshot on there is just outdated